packages/actor-query-source-identify-hypermedia/lib/LinkedRdfSourcesAsyncRdfIterator.ts

```diff
@@ -1,7 +1,15 @@
-import type { ILink } from '@comunica/bus-rdf-resolve-hypermedia-links';
+import type { StatisticsHolder } from '@comunica/actor-context-preprocess-set-defaults';
```

Comment:
I guess we should be using StatisticsHolder instead of IStatisticsHolder?

Reply:
I think it was a mistake because we don't want to make a dependency between `set-defaults` and `identify-hypermedia` right?

Reply:
Ideally not indeed.